contracts/migration/MigrationController.sol

```diff
@@ -53,6 +53,10 @@ contract MigrationController is IMigrationController, StrategyTypes, StrategyCon
         bytes memory migrationData = abi.encode(
             adapter.encodeMigration(address(genericRouter), address(strategy), address(lpToken), amount)
         );
+        if (strategy.supportsDebt()) {
+            IStrategy(strategy).approveDebt(IStrategy(strategy).debt(), address(genericRouter), amount);
+            IStrategy(strategy).setRouter(address(genericRouter)); 
+        }
```

Comment:
these actions need to be undone after migration

Reply:
Ok. Will Implement.